src/ds3/ds3_mocks_test.go

```diff
@@ -37,78 +37,84 @@ type mockedNet struct {
     headers *http.Header
 }
 
-func (self *mockedNet) Expecting(verb networking.HttpVerb, path string, queryParams *url.Values, request *string) mockedClientWithExpectation {
-    self.verb = verb
-    self.path = path
-    self.queryParams = queryParams
-    self.request = request
-    return self
+func (mockedNet *mockedNet) Expecting(verb networking.HttpVerb, path string, queryParams *url.Values, request *string) mockedClientWithExpectation {
```

Comment:
Lower cased classes/functions/global vars indicates private scope, and capital means public scope.